akkor_et_al_2024/Advanced_Flash_Stripper.py

```diff
@@ -241,9 +241,16 @@ def initialize(blk, solver, outlvl):
 
         # packing parameters for the stripper (random packing)
         blk.stripper.del_component(blk.stripper.a)
-        blk.stripper.a = Param(initialize=250, doc="Specific area (m2/m3)")
+        blk.stripper.a = Param(
+            initialize=250, mutable=True, doc="Specific area (m2/m3)"
+        )
         blk.stripper.del_component(blk.stripper.void)
-        blk.stripper.void = Param(initialize=0.97, doc="Void fraction")
+        blk.stripper.void = Param(initialize=0.97, mutable=True, doc="Void fraction")
+        blk.stripper.del_component(blk.stripper.P_drop_z)
```

Comment:
Minor question - why are you deleting this (and on line 247 too)?

Reply:
These values should be different for the stripper column. In my experience, it is safer to redefine the parameter than to change its value during initialization. In the past I had some issues where the expressions referencing the parameters were not updated correctly.

Reply:
Deleting components will actually make things more of a mess doing it this way, as any references will remain but be hidden. You should be able to change the values of these without any issues, and if you do see problems that generally implies you are doing something wrong elsewhere.

Seeing as this is part of the an initialization routine, why are you changing the parameter values here? Setting parameter values is something that should be done whilst defining the the model (i.e. before you start initialization), not when initializing it.